fake.go

```diff
@@ -519,6 +520,59 @@ func (fake *Fake) Dump() string {
 	return buf.String()
 }
 
+// ParseDump can parse a dump for a given nft instance.
+// It expects table name and the same nftables family in all rules based on a given fake.nftContext.
+// The best way to verify that everything important was properly parsed is to
+// compare given data with nft.Dump() output.
+func (fake *Fake) ParseDump(data string) (err error) {
+	lines := strings.Split(data, "\n")
+	var i int
+	var line string
+	parsingDone := false
+	defer func() {
+		if err != nil && !parsingDone {
+			err = fmt.Errorf("line %v=`%s`: %w", i+1, line, err)
```

Comment:
hm... the line is context information. the wrapped error should come first:

```suggestion
			err = fmt.Errorf("%w (at line %v: %s", err, i+1, line)
```